.github/workflows/sdk-assets-component-component.yml

```diff
@@ -51,3 +51,23 @@ jobs:
       with:
         name: component
         path: sdk/assets/component
+
+    - name: Send IcM on failure
+      if: ${{ failure() && (github.ref == 'refs/heads/main' || github.ref == 'ref/heads/release') }}
+      uses: kdestin/generate-icm@v1
```

Comment:
Dependency on the personal repo for any actions for validation is not recommended.